docker-compose-dev.yml

```diff
@@ -54,3 +71,7 @@ services:
       - POSTGRES_USER=nanocloud
     network_mode: host
     container_name: "postgres"
+
+volumes:
```

Comment:
We should use file sytem mounted volumes in developement to be able to access any file any time.